Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM --platform=$BUILDPLATFORM golang:1.19.0-bullseye as builder
```

Comment:
why not use 1.20 for latest?

Reply:
Yeah, good point. This probably needs to be updated in all of the AWS CSI driver repos.